apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue

```diff
@@ -69,7 +69,7 @@ const initSelectedVersion = () => {
         const originVersion = collectorFormState.originCollector?.plugin_info?.version;
         collectorFormStore.$patch({
             version: originVersion ?? collectorFormState.versions[0] ?? '',
-            autoUpgrade: originAutoUpgrade ?? true,
+            autoUpgrade: props.getVersionsOnPluginIdChange ? true : (originAutoUpgrade ?? true),
```

Comment:
question: Why is this line required? 😮

Reply:
This is because when you initialize createPage, autoUpgrade must be 'true'.

Reply:
This is because the `autoUpgrade` initial state of the version form on the create page must be `true`